Uppsala/UppEt12.xml

```diff
@@ -0,0 +1,885 @@
+                                </item>
+                                
+                                <item xml:id="a3">
+                                    <locus target="#125v"/>
+                                    <desc>Historical note on an event that took place in between <date from="1636" to="1646">1636 and 1646</date>.
+                                        For an edition (with partly different readings than what is presented below), 
+                                        an Italian translation, and a discussion, including of the date, 
+                                        see <bibl><ptr target="bm:Cerulli1956Upsala"/><citedRange unit="page">175–178</citedRange></bibl>.</desc>
+                                    <q>
+                                        ንጽሕፍ፡ ለክሙ፡ ዝክረ፡ ነገር፡ ዘኮነ፡ በመዋዕለ፡ ማቴዎስ፡ ወንጌላዊ፡ <sic resp="JK">ዘገበሩነ፡</sic>
+                                        ዘመናውያን፡ ሮም፡ ወኀርመን፡ ወግብጽ፡ <sic resp="JK">ዘይሰመ፡</sic> ኔሩዝ፡ በዘመነ፡ 
+                                        ማቴዎስ፡ ጳጳስ፡ ወማርቆስ፡ አቡን፡ <add place="right">በኢትዮጵያ፡</add> ወዮሐ<add place="above">ን</add>ስ፡ 
+                                        <choice><sic>ቄአቡን፡</sic><corr resp="PRS3076Cerulli">ቄሰገበዝ፡</corr></choice> በኢየሩሳሌም፡ 
+                                        ፭፡ ዐመት<supplied reason="omitted">፡</supplied> በበ፪፡ ፻፡ ኔሩዝ፡ ይበልዕ፡ 
+                                        ዘ<add place="above">መ</add>ጻእነ። ዕምኢትዮጵያ፡ ነሢአነ፡ ፍተ፡ ኅብስት፡ ኢትርስዑነ፡ 
+                                    ዘንተ፨፨። ዕምብርሃነ፡ ጸጋ፡ አውፅኡኒ፡ በይዕቲ፡ ዘመን፡ በብዝሀ፡ ወርቅ 
+                                    (<ref type="pers" corresp="PRS3076Cerulli"/> reads: ሰርቅ)</q>
+                                </item>
+                            </list>
+                        </additions>
+                        
+                        <bindingDesc>
+                            <binding contemporary="true" xml:id="binding">
+                                <decoNote xml:id="b1" type="Boards">Wooden boards covered with decorated leather.</decoNote>
+                                <decoNote xml:id="b2" type="EndLeaves" pastedown="L R UTI">Blue patterned textile pastedowns.</decoNote>
+                                <decoNote xml:id="b3" type="Fastening">Defect metallic cramps.</decoNote>
+                            </binding>
+                        </bindingDesc>
+                        
+                    </physDesc>
+                                        
+                    <history>
+                        <origin>
+                            <origDate notBefore="1563" notAfter="1597">1563–1697</origDate>
+                        </origin>
+                        <provenance>
+                                The ultimate origin of the manuscript is unknown. According to the colophon on 
+                                <locus from="123ra" to="123rb"/>, the manuscript was copied by 
+                            <ref type="pers" corresp="PRS14712MazgabaDawit"/> by the order of his teacher 
```

Comment:
`<persName ref="PRS14712MazgabaDawit"/>` etc, same for placeName later

Reply:
just so I learn, do you know when the encoding I used should be used?

Reply:
So then `<ref type="pers">` has no function?

Reply:
I believe so